scm/content.go

```diff
@@ -16,10 +18,11 @@ type (
 	// ContentParams provide parameters for creating and
 	// updating repository content.
 	ContentParams struct {
-		Ref     string
```

Comment:
the `Ref` attribute should not be removed. It is a placeholder that is required for other providers.